cudamapper/CMakeLists.txt

```diff
@@ -13,6 +13,22 @@ project(cudamapper)
 # Process data subdirectory first
 add_subdirectory(data)
 
+execute_process(COMMAND
+  git describe --tag --dirty
+  WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
+  OUTPUT_VARIABLE VERSION
+  OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+string(REGEX REPLACE "^v([0-9]+)\\..*" "\\1" VERSION_MAJOR "${VERSION}")
```

Comment:
version major, minor and patch should come from the version number that's checked into $TOP/VERSION (https://github.com/clara-parabricks/ClaraGenomicsAnalysis/blob/dev-v0.5.0/VERSION). And then version_SHA1 can come from git

Reply:
How about we remove $TOP/version and use top git tag?